PSC-ONBOARDING-OFFBOARDING.md

```diff
@@ -0,0 +1,171 @@
+[joining](https://github.com/kubernetes/security/blob/master/security-release-process.md#joining)
+for further details on the PSC selection process.
+
+This selection process is initiated with an email sent to the
+[kubernetes-security-discuss mailing list](https://groups.google.com/forum/#!forum/kubernetes-security-discuss)
+once the three month Associate Role period is complete.
+
+The following template may be used:
+
+```
+I want to nominate Jane Doe to join the Product Security Committee as
+a full member. She has been an associate member since December 4th 2018
+and in that role over the last three months he has helped on a number of
+tasks including:
+
+<list work carried out as associate>
+
+Our process for accepting this nomination is lazy consensus and I will
+close the round of consensus on March 15th, 2019 to leave time for any
+comments or question. If the nomination is accepted Jane will take on
+the full responsibilities of a Security Committee member and take on a
+weekly rotation for security@kubernetes.io on-call rotation.
+```
+
+### Required file and access grant updates.
+
+Once the PSC has promoted an individual(s)  to a full PSC member, various
+systems require updating. Likewise, should a PSC member leave the PSC - the
+reverse is required, with the user being removed from each system.
+
+#### kubernetes/community repository
+
+##### file: https://github.com/kubernetes/community/blob/master/sigs.yaml
+
+Add / remove the PSC member(s)  github name from `community/sigs.yaml` to the
+existing `label`| `leadership` field, according to the usernames alphabetical
+placing:
+
+```yaml
+label: product-security
+  leadership:
+    chairs:
+    - github: jdoe
+      name: Jane Doe
+      company: Acme LTD
+```
+
+##### file: https://github.com/kubernetes/community/blob/master/OWNERS_ALIASES
+
+Add / remove the PSC member(s) github name from `community/OWNERS_ALIASES` to the
+existing `committee-product-security` field, according to the usernames
+alphabetical placing:
+
+```yaml
+committee-product-security:
+   - jdoe
+```
+
+##### file: https://github.com/kubernetes/community/blob/master/committee-product-security/README.md
+
+Add / remove the PSC member(s) github name from `community/committee-product-security/README.md`
+to the existing `committee-product-security` field, according to the usernames
+alphabetical placing:
+
+```markdown
+## Members
+
+* Joel Doe (**[@jdoe](https://github.com/jdoe)**), ACME Ltd
+```
+
+##### file: https://github.com/kubernetes/community/blob/master/sig-list.md
+
+Add / remove the PSC member(s) github name from `community/sig-list.md`
+to the existing `committee-product-security` field, according to the usernames
+alphabetical placing:
+
+```markdown
+|[Product Security](committee-product-security/README.md)|product-security|* [Jane Doe](https://github.com/jdoe), ACME Ltd<br>* [Private Mailing List](security@kubernetes.io)
+```
+#### kubernetes/security repository
```

Comment:
Can we put this section first, and have the kubernetes/security repo be the source of truth? I.e. discussion & approval should happen on the k8s/security PR, and then all the other changes should be considered approved-by-default after that merges.

Reply:
done and added some text about changes must be approved / merged in /security first, before /community